src/traits.d

```diff
@@ -951,6 +951,8 @@ extern (C++) Expression semanticTraits(TraitsExp e, Scope* sc)
                 }
                 if (sm.isTypeInfoDeclaration()) // Bugzilla 15177
                     return 0;
+                if (sds.isClassDeclaration() && strcmp(sm.kind(), "import") == 0)
```

Comment:
try to not use kind,
There should be a either an op field you can query or an is method.